src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: $white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__product-name,
+  &__price-value {
+    @include font-weight-500;
+
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    color: $dark-blue;
+  }
+
+  &__price-value,
+  &__buy {
+    @include font-weight-700;
+
+    font-size: 16px;
+  }
+
+  &__product-code,
+  &__reviews,
+  &__product-price {
+    @include font-weight-400;
+
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    width: 100%;
+  }
+
+  &__reviews {
+    color: $dark-blue;
+  }
+
+  &__product-price {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__rating,
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    width: 100%;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    background-color: $blue;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    color: $white;
+    text-transform: uppercase;
+    font-size: 14px;
+    line-height: 16px;
+    border: 1px solid $blue;
+
+    &:hover {
+      background-color: $white;
+      color: $blue;
+    }
+  }
+
+  &__rating,
+  &__buy {
+    margin-top: 16px;
+  }
```

Comment:
The `&__rating` and `&__buy` selectors are repeated in two different blocks. Consider consolidating these styles to avoid redundancy and potential conflicts.